src/main/scala/kinesis/mock/models/Streams.scala

```diff
@@ -8,44 +8,37 @@ import cats.syntax.all._
 import io.circe._
 import io.circe.derivation._
 
-final case class Streams(streams: SortedMap[StreamName, StreamData]) {
+final case class Streams(streams: SortedMap[StreamArn, StreamData]) {
```

Comment:
Stream data is now keyed by the StreamArn rather than the StreamName, allowing for multi-region separation.